Integrations/integration-McAfeeWebGateway.yml

```diff
@@ -64,6 +64,9 @@ script:
         );
 
         if (res.StatusCode < 200 || res.StatusCode >= 300) {
+            if (res.Status.indexOf('pass=') !== -1) {
+                throw 'Request Failed.\nStatus code: ' + res.StatusCode + '. Check your Server URL and Port.';
+            }
             throw 'Request Failed.\nStatus code: ' + res.StatusCode + '.\nBody: ' + JSON.stringify(res) + '.';
```

Comment:
take a look at the example of the bad test print you putted in the original issue.
there is no "Body:" there. so where did it came from?